src/tests/electron/common/view-controllers/view-controller.ts

```diff
@@ -52,6 +52,13 @@ export abstract class ViewController {
         return await this.screenshotOnError(async () => this.client.isEnabled(selector));
     }
 
+    public async itemTextContainsTarget(selector: string, target: string): Promise<boolean> {
+        return await this.screenshotOnError(async () => {
+            const itemText: string = await this.client.getText(selector);
+            return itemText.indexOf(target) >= 0;
```

Comment:
we can use the [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) method instead if it feels more readable